scripts/events.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var proNavsAndFilters = { };
+
+proNavsAndFilters.mainNavListener = function() {
+	$('.main_nav').on('click', '.nav_link', function(){
+		$('.page-section').hide();
+		$(`#${$(this).data('nav')}`).fadeIn(1000);
+	});
+	$('.main_nav .nav_link:first').click();
+}
+
+proNavsAndFilters.fillTheFilters = function() {
```

Comment:
If you want to implement filtering, to populate your filter list, you could do something like this:
```
projects.forEach(function(project) {
	$('#type-filter').append('<option>' + project.projectType + '</option>')
})
```

You'll have to add projectType as a property on your constructor, though. Then, you'll have to implement the 'hide and show' functionality.